package.json

```diff
@@ -14,6 +14,8 @@
     "build/",
     "doc/",
     "lib/",
+    "patches/",
+    "*.md",
```

Comment:
The `patches` folder was also missing, and I've added `*.md` too. Without `*.md`, by default npm publishes the CHANGELOG, README and LICENSE files, and with `*.md` we further publish the INSTALL and TROUBLESHOOTING files, which I thought could help someone with installation issues (if they are browsing that folder, chances are they are trying to solve some issue...).